### PR TITLE
Typo correction

### PR DESCRIPTION
**Pull request by AanshSavla**
Fixed typos
Descritpion -> Description
Descritption -> Description
Moudle -> Module

---
**^Add meaningful description above**

By opening a pull request you confirm that, unless explicitly stated otherwise, the changes -

 - are all your own work, and you have the right to contribute them.
 - are contributed solely under the terms and conditions of the Apache License 2.0 (see section 5 of the license for more information).

Please make sure (eg. `git log`) that all commits have a valid name and email address for you in the Author field.

If you're a first time contributor, see the Contributing guidelines for more information.

If you're a committer, please label the PR before pressing "Create pull request" so that the right test jobs can run.

### CODE REVIEW

File: java/java.navigation/src/org/netbeans/modules/java/navigation/ElementNode.java [hunk 1]
```diff
@@ -309,7 +309,7 @@ public void updateRecursively( Description newDescription ) {
         }
     }
     
-    public Description getDescritption() {
+    public Description getDescription() {
```
Comment (mbien) [inline java/java.navigation/src/org/netbeans/modules/java/navigation/ElementNode.java:312]: note to self: checked and `java.navigation` does not export this package (or any other package). `Description` itself is package private interestingly.

@matthiasblaesing can we make this method package private?

File: java/refactoring.java/src/org/netbeans/modules/refactoring/java/ui/elements/ElementNode.java [hunk 2]
```diff
@@ -219,7 +219,7 @@ public void selectionChanged() {
         fireIconChange(); // TODO: Strange
     }
 
-    public Description getDescritption() {
+    public Description getDescription() {
```
Comment (mbien) [inline java/refactoring.java/src/org/netbeans/modules/refactoring/java/ui/elements/ElementNode.java:222]: note to self: similar here, package is not exported. However the difference is that `Description` is public.

File: platform/o.n.core/src/org/netbeans/core/ui/SwingBrowser.java [hunk 3]
```diff
@@ -51,7 +51,7 @@ private void init () {
     /** Getter for browser name
      *  @return browserName name of browser
      */
-    public String getDescritpion () {
+    public String getDescription () {
```
Comment (mbien) [inline platform/o.n.core/src/org/netbeans/core/ui/SwingBrowser.java:54]: note to self: we are lucky here too, package not exported.

**Review by mbien — CHANGES_REQUESTED** [review]
commits here still contain an invalid email, you can see it here:
https://patch-diff.githubusercontent.com/raw/apache/netbeans/pull/5583.patch

would need fixing before this can be integrated

**Comment by AanshSavla** [comment]
> commits here still contain an invalid email, you can see it here: https://patch-diff.githubusercontent.com/raw/apache/netbeans/pull/5583.patch
> 
> would need fixing before this can be integrated

@mbien Can you suggest where can I make these changes?

**Comment by mbien** [comment]
@AanshSavla I already did on the first PR you created https://github.com/apache/netbeans/pull/5577#pullrequestreview-1318033729

**Comment by AanshSavla** [comment]
> @AanshSavla I already did on the first PR you created [#5577 (review)](https://github.com/apache/netbeans/pull/5577#pullrequestreview-1318033729)

The problem was in my git config. I just changed it in local repo. But figuring out that how to push it on remote repo.

**Review by mbien — APPROVED** [review]
I squashed the commits for you. Going to merge in a day or two assuming the tests are green.

**Comment by tbw777** [comment]
@AanshSavla 
shoudl 18 matches

**Comment by mbien** [comment]
merging. @AanshSavla congrats to your first contribution!

**Comment by AanshSavla** [comment]
Thank You! @mbien

**Pull request merged**